src/Type/Definition/DefinitionInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Digia\GraphQL\Type\Definition;
+
+/**
+ * Tagging interface for GraphQL definitions (schema, types, directive, etc.).
+ */
+interface DefinitionInterface
+{
+}
```

Comment:
Introduce this tagging interface to be able to type-hint classes within the `DefinitionPrinter`